### PR TITLE
Experimental: M5Stack Support (Cont)

### PR DESCRIPTION
**Pull request by chopmann**
+ Fixed case-sensitie board name (Linux)
+ Added M5Stack Fire board
+ WiFi sniffing tested with verbose mode (NO LoRa or GPS)

### CODE REVIEW

File: src/hal/m5fire.h [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+// user defined sensors
+//#define HAS_SENSORS 1 // comment out if device has user defined sensors
+
+#define CFG_sx1276_radio 1 // select LoRa chip
+#define BOARD_HAS_PSRAM // use if board has external PSRAM
+#define DISABLE_BROWNOUT 1 // comment out if you want to keep brownout feature
+
+//#define HAS_DISPLAY 1
+#define HAS_TFT 1
```
Comment (chopmann) [inline src/hal/m5fire.h]: Dont know how you want to implement the feature.
I think you also need to specify the Controller (ILI9341)

File: src/hal/m5fire.h [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+//#define DISPLAY_FLIP  1 // use if display is rotated
+//#define BAT_MEASURE_ADC ADC1_GPIO35_CHANNEL // battery probe GPIO pin -> ADC1_CHANNEL_7
+//#define BAT_VOLTAGE_DIVIDER 2 // voltage divider 100k/100k on board
+
+#define HAS_LED NOT_A_PIN // no on board LED (?)
```
Comment (chopmann) [inline src/hal/m5fire.h]: SK6812 3535 RGB LED x 10

File: src/hal/m5fire.h [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+#define LORA_MISO MISO
+#define LORA_MOSI MOSI
+#define LORA_RST  GPIO_NUM_36
+#define LORA_IRQ  GPIO_NUM_26
+#define LORA_IO1  GPIO_NUM_34 // must be externally wired on PCB!
+#define LORA_IO2  LMIC_UNUSED_PIN
+
+
+// enable only if you want to store a local paxcount table on the device
+#define HAS_SDCARD  1      // this board has an SD-card-reader/writer
```
Comment (chopmann) [inline src/hal/m5fire.h]: @cyberman54 Don't approve just yet. Need to find a suitable card.

File: src/hal/m5fire.h (hunk 2 above)
Comment (cyberman54) [reply src/hal/m5fire.h]: SK6812 is supported by [SmartLEDs lib](https://github.com/RoboticsBrno/SmartLeds). So should work to activate it here as RGB_LED, look in lopy4.h

#define HAS_RGB_LED SmartLed rgb_led(LED_SK8612, 10, GPIO_NUM_?) // GPIO=?

File: src/hal/m5fire.h (hunk 1 above)
Comment (cyberman54) [reply src/hal/m5fire.h]: Yes, we need some way to specify the controller here. Maybe like GPS_SERIAL, look in generic.h

File: src/hal/m5fire.h (hunk 3 above)
Comment (cyberman54) [reply src/hal/m5fire.h]: ok

File: src/hal/m5fire.h (hunk 2 above)
Comment (cyberman54) [reply src/hal/m5fire.h]: Does the M5 fire really have 10 RGB LEDs?

Comment (cyberman54) [reply src/hal/m5fire.h]: Yes, it has.

**Pull request merged**